python-lib/nlp_utils.py

```diff
@@ -3,16 +3,37 @@
 import unicodedata
 
 
-def get_keyword(text: AnyStr, normalize_case: bool) -> AnyStr:
+def get_text_case(text: AnyStr, normalize_case: bool) -> AnyStr:
```

Comment:
I am not sure the function name conveys what it does. Perhaps "normalize_case"?

Reply:
good idea :) 0765d3c